it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/storage/conditions/GCSArtifactsCheck.java

```diff
@@ -38,6 +40,9 @@ public abstract class GCSArtifactsCheck extends ConditionCheck {
   @Nullable
   abstract Integer maxSize();
 
+  @Nullable
+  abstract String tableName();
```

Comment:
If we are writing a generic GCS checker - then this should be replaced with something generic- like a pattern to check. As such table name has no meaning for a GCS artifact.

Reply:
makes sense , updated PTAL